api/src/test/groovy/com/okta/sdk/http/HttpRequestsTest.groovy

```diff
@@ -13,14 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.okta.sdk.impl.http;
+package com.okta.sdk.http
 
-import com.okta.sdk.impl.config.ClientConfiguration;
+import org.testng.annotations.Test
 
-/**
- * @since 1.2.0
- */
-public interface RequestExecutorFactory {
+import static org.testng.Assert.assertEquals
+import static org.testng.Assert.assertNotNull
+import static org.testng.Assert.assertTrue
+import static org.testng.Assert.fail
+
+class HttpRequestsTest {
```

Comment:
Are you planning to add tests here or can this be removed?

Reply:
Removed, those tests were moved to the `impl` project due to reflection being used